python/tvm/relay/frontend/mxnet.py

```diff
@@ -444,6 +444,15 @@ def _mx_tile(inputs, attrs):
     return _op.tile(inputs[0], **new_attrs)
 
 
+def _mx_take(inputs, attrs):
+    assert len(inputs) == 2
+    mode = attrs.get_str("mode", "clip")
+    if mode == "raise":
+        raise RuntimeError("take doesn't support raise mode")
```

Comment:
enable the support?

Reply:
In fact, currently mxnet doesn't support raise either. 
http://mxnet.incubator.apache.org/api/python/ndarray/ndarray.html?highlight=take#mxnet.ndarray.take